src/components/QueryEditor/QueryEditor.test.tsx

```diff
@@ -3,7 +3,47 @@ import { render, screen, fireEvent, act, waitFor } from '@testing-library/react'
 import { QueryEditor } from './QueryEditor';
 import { Group, Region, XrayJsonData, XrayQuery, XrayQueryType } from '../../types';
 import { XrayDataSource } from '../../DataSource';
-import { DataSourceInstanceSettings } from '@grafana/data';
+import { DataSourceInstanceSettings, ScopedVars, VariableModel } from '@grafana/data';
+import { TemplateSrv } from '@grafana/runtime';
+
+jest.mock('@grafana/runtime', () => {
```

Comment:
it seems like you're updating a mock but not any tests in this file. Is that intentional?